### PR TITLE
Migrate lambda sample dag to system test (AIP-47)

### PR DESCRIPTION
**Pull request by vincbeck**
Migrate lambda sample dag to system test

### CODE REVIEW

**Comment by potiuk** [comment]
Tests failing

**Comment by vincbeck** [comment]
Yep, currently working on it :)

**Comment by vincbeck** [comment]
It's now fixed

File: tests/system/providers/amazon/aws/example_lambda.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import json
+import zipfile
+from datetime import datetime
+
+import boto3
+
+from airflow import models
+from airflow.decorators import task
+from airflow.models.baseoperator import chain
+from airflow.providers.amazon.aws.operators.aws_lambda import AwsLambdaInvokeFunctionOperator
```
Comment (eladkal) [inline tests/system/providers/amazon/aws/example_lambda.py:27]: I thought we handled all the `aws_` prefixes in file names? @ferruzzi ?

**Review by eladkal — APPROVED** [review]

File: tests/system/providers/amazon/aws/example_lambda.py (hunk 1 above)
Comment (eladkal) [reply tests/system/providers/amazon/aws/example_lambda.py:27]: https://github.com/apache/airflow/pull/25100

Comment (ferruzzi) [reply tests/system/providers/amazon/aws/example_lambda.py:27]: Looking at the Git Blame, looks like this was added after that sweep.  Thanks for the fix.

**Pull request merged**